Terraform Registry publishing FAQ (#223)
hashicorp/terraform-docs-common: website/docs/registry/modules/publish.mdx

under the "Manage Module" dropdown. This process may take a few minutes.
Please only do this if you do not see the version appear, since it will
cause the registry to resync _all versions_ of your module.

## Removing published content

> The owner of a module is able to use the Terraform Registry to remove one or all versions of the module. Please be aware that [we recommend against removing anything from the Registry unless it contains a critical flaw](/terraform/registry/faq#how-do-i-remove-a-version-of-my-provider-module-or-policy-library-that-has-already-been-published).

Reviewer: ```suggestion
The owner of a module can use the Terraform Registry to remove one or all versions of that module. Please be aware that [we do not recommend removing anything from the Registry unless it contains a critical flaw](/terraform/registry/faq#how-do-i-remove-a-version-of-my-provider-module-or-policy-library-that-has-already-been-published).
```

(May need to update link if header changes)

> To do this, browse to your module's page and select the appropriate option from the "Manage Module" dropdown menu.

Reviewer: ```suggestion
To remove a module version, navigate to your module's page and select the appropriate option from the **Manage Module** dropdown menu.
```
